Fill.py

```diff
@@ -15,8 +17,57 @@ class FillError(RuntimeError):
     pass
 
 
-def _log_fill_progress(name: str, placed: int, total_items: int) -> None:
-    logging.info(f"Current fill step ({name}) at {placed}/{total_items} items placed.")
+class FillLogger():
+    min_size: int   = 1000
+    min_time: float = 0.25
+
+    def __init__(self, total_items: int):
+        self.start_time       = time.time()
+        self.prev_time        = time.time()
+        self.cur_time         = time.time()
+        self.step: int        = max(round(total_items * 0.1), 1000)
+        self.total_items: int = total_items
```

Comment:
PEP8 says this:
```python
# Correct:
x = 1
y = 2
long_variable = 3

# Wrong:
x             = 1
y             = 2
long_variable = 3
```

Reply:
I'll add this in to other portions of a commit that's coming soon. Thanks for pointing it out.

Reply:
BTW, I hope my message here is not interpreted as "You should do it this way" - just pointing out that it makes an inconsistency with what's in the AP style guide (which is basically PEP8 with some noted exceptions).

I would not not be against a change to the AP style guide to say this is ok, if there's a discussion and the people who work in core decide it's ok.

Reply:
Until AP decides to change the style guide I should absolutely attempt to adhere to it. I've updated this and it should now conform. If you see anything else please point it out! I've run pylint against just this chunk of code (I don't feel like it's my job to test the entire file against PEP8), but it didn't catch this exception so extra help is appreciated.